datamodel/initdb.d/03_dcsa_im_v3_0.sql

```diff
@@ -275,9 +275,12 @@ CREATE TABLE dcsa_im_v3_0.booking (
     vessel_id uuid NULL REFERENCES dcsa_im_v3_0.vessel(id),
     export_voyage_number varchar(50) NULL,
     place_of_issue varchar(100) NULL REFERENCES dcsa_im_v3_0.location(id),
-    pre_carriage_mode_of_transport_code varchar(3) NULL REFERENCES dcsa_im_v3_0.mode_of_transport(mode_of_transport_code)
+    pre_carriage_mode_of_transport_code varchar(3) NULL REFERENCES dcsa_im_v3_0.mode_of_transport(mode_of_transport_code),
+    valid_until timestamp with time zone NULL
 );
 
+CREATE UNIQUE INDEX unq_valid_until_booking_idx ON dcsa_im_v3_0.booking(valid_until) WHERE valid_until IS NULL;
```

Comment:
I think we want the booking reference to be unique, not the valid_until date (which is guaranteed to be NULL)